lib/code_climate.ex

```diff
@@ -43,7 +129,7 @@ defmodule CodeClimate do
   defp process_exclude_paths(opts, %{"exclude_paths" => exclude_paths}, path) do
```

Comment:
Do we still expect plugins to process exclude paths? The CLI hasn't sent them down to plugins in like 4 years.

Reply:
Good catch - thanks!